### PR TITLE
Add bulk loading Python Sample

### PR DESCRIPTION
**Pull request by tonioshikanlu**

### CODE REVIEW

**Comment by tonioshikanlu** [comment]
Hey Johnathan, can you look at this!

**Comment by tonioshikanlu** [comment]
@jsimonweb Please, review. Thanks!

File: spanner/cloud-client/bulk_load_csv/batch_import.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+
+import csv
+import time
+import threading
+import argparse
+from google.cloud import spanner
+
+
+def is_bool_null(file):
```
Comment (gguuss) [inline spanner/cloud-client/bulk_load_csv/batch_import.py]: Maybe add a unit test for this?

File: spanner/cloud-client/bulk_load_csv/batch_import.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            # Changes the string to boolean.
+            if data[line][cell] == 'true':
+                data[line][cell] = eval('True')
+            # Changes blank string to python readable None type.
+            if data[line][cell] == '':
+                data[line][cell] = None
+    return (data)
+
+
+def divide_chunks(lst, n):
```
Comment (gguuss) [inline spanner/cloud-client/bulk_load_csv/batch_import.py]: Unit test?

File: spanner/cloud-client/bulk_load_csv/schema.ddl [hunk 3]
```diff
@@ -0,0 +1,35 @@
+CREATE TABLE comments (
```
Comment (gguuss) [inline spanner/cloud-client/bulk_load_csv/schema.ddl]: Copyright header?

File: spanner/cloud-client/bulk_load_csv/batch_import_test.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+"""Test for batch_import"""
```
Comment (gguuss) [inline spanner/cloud-client/bulk_load_csv/batch_import_test.py]: Missing [copyright header](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#license-header).

Comment (tonioshikanlu) [reply spanner/cloud-client/bulk_load_csv/batch_import_test.py]: Done

File: spanner/cloud-client/bulk_load_csv/schema.ddl (hunk 3 above)
Comment (tonioshikanlu) [reply spanner/cloud-client/bulk_load_csv/schema.ddl]: Done

File: spanner/cloud-client/bulk_load_csv/batch_import.py (hunk 2 above)
Comment (tonioshikanlu) [reply spanner/cloud-client/bulk_load_csv/batch_import.py]: Done

File: spanner/cloud-client/bulk_load_csv/batch_import.py (hunk 1 above)
Comment (tonioshikanlu) [reply spanner/cloud-client/bulk_load_csv/batch_import.py]: Done

**Pull request merged**